src/components/img.ts

```diff
@@ -26,18 +30,45 @@ const styles = css`
     `}
 `;
 
-const Img: FC<Props> = ({ image, sizes, imageMappings, className }) =>
-    styledH('img', {
+const mediaQuery = (dpr: Dpr): string => {
+    switch (dpr) {
+        case Dpr.Two:
+            return '(-webkit-min-device-pixel-ratio: 1.25), (min-resolution: 120dpi)';
+        case Dpr.One:
+        default:
+            return '';
+    }
+};
+
+const Source: FC<SourceProps> = ({ sizes, srcSet, dpr }) =>
+    h('source', {
         sizes,
-        srcSet: srcset(image.src, imageMappings),
-        src: src(imageMappings, image.src, 500),
-        alt: image.alt.withDefault(''),
-        className: 'js-launch-slideshow',
-        css: [styles, className],
-        'data-caption': image.nativeCaption.withDefault(''),
-        'data-credit': image.credit.withDefault(''),
+        srcSet,
+        media: mediaQuery(dpr),
     });
 
+const Img: FC<Props> = ({ image, sizes, className }) =>
+    h('picture', null, [
+        h(Source, {
+            sizes,
+            srcSet: image.dpr2Srcset,
+            dpr: Dpr.Two,
+        }),
+        h(Source, {
+            sizes,
+            srcSet: image.srcset,
+            dpr: Dpr.One,
+        }),
```

Comment:
It occurred to me that this could actually be simpler if we inlined it rather than creating the `Source` component:

```suggestion
        h('source', {
            sizes,
            srcSet: image.dpr2Srcset,
            media: '(-webkit-min-device-pixel-ratio: 1.25), (min-resolution: 120dpi)',
        }),
        h('source', {
            sizes,
            srcSet: image.srcset,
        }),
```

That way we can drop the `Source` component, the `mediaQuery` function, _and_ we no longer need import `Dpr`.